packages/thirdweb/src/indexer/endpoints/getReceipts.ts

```diff
@@ -0,0 +1,6 @@
+// TODO clarify the types, no need to sanitize here because we are already doing it server-side?
+export async function getReceipts(client: any, params: any) {
+  const blocksResponse = await client(params);
+  // TODO: Calrify structured return type through the indexer
+  return await blocksResponse.json();
```

Comment:
at first thought this should (maybe?) return the same type as the RPC method? that way it can be used interchangeably? Maybe there's additional optional fields?

Reply:
Converted transactions and blocks to the closest possible formats with extended data